addons/mass_mailing/wizard/mailing_contact_import.py

```diff
@@ -91,12 +91,25 @@ def action_import(self):
 
         if ignored := len(contacts) - len(unique_contacts):
             message = _(
-                "Contacts successfully imported. Number of contacts imported: %(imported_count)s. Number of duplicates ignored: %(duplicate_count)s",
+                "%(imported_count)s contacts have been imported. %(duplicate_count)s contacts are duplicates and have been ignored.",
                 imported_count=len(unique_contacts),
                 duplicate_count=ignored,
             )
         else:
-            message = _("Contacts successfully imported. Number of contacts imported: %(imported_count)s", imported_count=len(unique_contacts))
+            message = _("%(imported_count)s contacts have been imported.", imported_count=len(unique_contacts))
+
+        if self.env.context.get('no_redirect', False):
```

Comment:
None default is sufficient for the if
```suggestion
        if self.env.context.get('no_redirect'):
```